config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -189,7 +189,7 @@ periodics:
       - --deployment=node
       - --gcp-project-type=node-e2e-project
       - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/perf-image-config.yaml
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml
```

Comment:
should [this tab](https://testgrid.k8s.io/sig-node-kubelet#node-kubelet-flaky) repel the "Node Performance Testing" tests using the `skip`? I am very confused why this task picks up performance tests. 

This definition says: `- --test_args=--nodes=1 --focus="\[Flaky\]"`

And name of the test doesn't include "Flaky":

```
E2eNode Suite.[sig-node] Node Performance Testing [Serial] [Slow] Run node performance testing with pre-defined workloads NAS parallel benchmark (NPB) suite - Embarrassingly Parallel (EP) workload [cos-stable1]E2eNode Suite.[sig-node] Node Performance Testing [Serial] [Slow] Run node performance testing with pre-defined workloads NAS parallel benchmark (NPB) suite - Embarrassingly Parallel (EP) workload [ubuntu]E2eNode Suite.[sig-node] Node Performance Testing [Serial] [Slow] Run node performance testing with pre-defined workloads NAS parallel benchmark (NPB) suite - Integer Sort (IS) workload [cos-stable1]E2eNode Suite.[sig-node] Node Performance Testing [Serial] [Slow] Run node performance testing with pre-defined workloads NAS parallel benchmark (NPB) suite - Integer Sort (IS) workload [ubuntu] 
```